corehq/tests/pytest_plugins/timelimit.py

```diff
@@ -53,6 +53,11 @@ def enforce_limit(self, item, event):
             raise AssertionError(f"{event} time limit ({limit}) exceeded: {duration}")
         self.limits = None
 
+    def time(self):
+        if escape_hatch.is_travelling():
+            return escape_hatch.time.time()
+        return time.time()
```

Comment:
Why not this? I assume that works the same regardless of if we are time travelling or not.
```suggestion
        return escape_hatch.time.time()
```